distributions/__init__.py

```diff
@@ -36,6 +36,7 @@
 from .Dist_yz4244 import Dist_yz4244
 from .Dist_speedreed import Dist_speedreed
 from .Dist_speedreed2 import Dist_speedreed2
+from .Dist_yl6709 import Dist_yl6709
```

Comment:
You will want to do this in a different branch for testing. If I merge this it will cause conflicts for others.
Can you remove this line.